devices/SRS_830.py

```diff
@@ -4,6 +4,7 @@
 
 
 class SRS830(BaseDevice):
+    _SRS_lock_in_data_key = "channel ID"
```

Comment:
Why do we add this?

Reply:
I added this here because something similar was added in the NI_DAQ script, so I was trying to stay consistent. If you think it is unnecessary then I can remove it.
![image](https://github.com/Quantum-Science-and-Technology-Lab/labber-wrapper/assets/161180155/5e25e457-8e41-4212-a656-a8c13720ee01)